x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -65,6 +67,14 @@ export interface ConditionEntryInputProps {
   'data-test-subj'?: string;
 }
 
+// adding a style prop on EuiFlexGroup works only partially
+// and for some odd reason garbles up gridTemplateAreas entry
+const StyledEuiFlexGroup = styled(EuiFlexGroup)`
+  display: grid;
```

Comment:
Does not feel right to be using `EuiFlexGroup` and `EuiFlexItems`. I'm thinkin the only thing we're gaining is the padding/margins that come with Flex items from Eui. Also, by changing the display type, Im not sure anymore if all of the EuiFlexGroup/EuiFlexItem props still apply (maybe I think invlunce the FlexBox css).

I'm fine if we merge as is, but we shoudl rework this to not use FlexGroup since what we really want is grid.

Reply:
Okay so now I've removed `EuiFlexGroup` and `EuiFlexItem`s and used `div`s instead. 0a60598cbaaa6d3c9658f4977d64a87fac5cc946